xml/System.IO/Path.xml

```diff
@@ -1459,7 +1459,7 @@ Unlike the string overload, this method doesn't normalize directory separators.
       <Docs>
         <param name="path">The path from which to obtain root directory information.</param>
         <summary>Gets the root directory information of the specified path.</summary>
-        <returns>The root directory of <paramref name="path" />, or <see langword="null" /> if <paramref name="path" /> is <see langword="null" />, or an empty string if <paramref name="path" /> does not contain root directory information.</returns>
+        <returns>The root directory of <paramref name="path" />, or <see langword="null" /> if <paramref name="path" /> is <see langword="null" /> or only contains spaces, or an empty string if <paramref name="path" /> does not contain root directory information.</returns>
```

Comment:
The "or only contains spaces" is specific to Windows, no?

Reply:
Yes, that is correct. Only Windows.

Reply:
I will make sure to clarify that as well.